core/templates/dev/head/components/OutcomeEditorDirective.js

```diff
@@ -63,6 +67,21 @@ oppia.directive('outcomeEditor', [function() {
           onExternalSave();
         });
 
+        $scope.getCurrentInteractionId = function() {
```

Comment:
I think you passed it in as $scope.suppressWarnings() -- see the 'scope' object at the top of this directive. Does logging that display anything?